asf_heat_pump_suitability/pipeline/run_scripts/run_calculate_garden_size.py

```diff
@@ -112,9 +123,16 @@ def parse_arguments() -> argparse.Namespace:
             )
 
         # Prepare building footprints data
-        building_footprints_gdf = building_footprint.transform_gdf_building_footprints(
-            building_file
-        )
+        try:
```

Comment:
This `try`/`except` was added to deal with some of the Scottish `.shp` files which had some corrupted polygons.